_docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhd-frontend/src/styles/rhd-theme/components/_navigation.scss

```diff
@@ -302,6 +302,8 @@ $custom-nav-breakpoint: 1150px;
 
   .pf-c-brand {
     width: 100px;
+    display: inline-block;
```

Comment:
This centers the redhat logo in the black nav bar.  It was set by foundation previously.